VirtualList/VirtualList.js

```diff
@@ -417,10 +417,15 @@ VirtualList.propTypes = /** @lends sandstone/VirtualList.VirtualList.prototype *
 	scrollMode: PropTypes.string,
 
 	/**
-	 * Spotlight Id.
+	 * The container id for [Spotlight container]@link spotlight/SpotlightContainerDecorator/#SpotlightContainerDecorator.spotlightId}.
+	 *
+	 * It is required to restore focus after `VirtualList` remount.
+	 *
+	 * For example, when the `VirtualList` is within a [Panel]{@link sandstone/Panels.Panel},,
+	 * the `Spotlight` will store the id to restore focus when returning to the `Panel`.
```

Comment:
the `Spotlight` will store the `spotlightId` to restore focus when returning to the `Panel`.

Reply:
I fixed